github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1611,43 +1630,14 @@ presubmits:
       - command:
         - /usr/local/bin/runner.sh
         - /bin/sh
-        - -c
+        - -ce
         - |
-          # install yq
-          curl -Lo ./yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
-          chmod +x ./yq && mv ./yq /usr/local/bin/yq
-
-          # get kubectl
-          curl -L "https://dl.k8s.io/release/v1.19.7/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl
-          chmod +x /usr/local/bin/kubectl
-
-          # get kubeconfig
-          source ../project-infra/hack/manage-secrets.sh
-          decrypt_secrets
-          extract_secret 'kubeconfigARM' /kubeconfig
-
-          # login quay.io
-          cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
-
-          # run the test
           automation/test.sh
         env:
         - name: TARGET
-          value: external
-        - name: KUBEVIRT_PROVIDER
-          value: external
-        - name: DOCKER_PREFIX
-          value: quay.io/kubevirtci
-        - name: DOCKER_TAG
-          value: aarch64_test
-        - name: KUBECONFIG
-          value: /kubeconfig
-        - name: kubectl
-          value: /usr/local/bin/kubectl
-        - name: IMAGE_PULL_POLICY
-          value: Always
-        - name: BUILD_ARCH
-          value: crossbuild-aarch64
+          value: kind-1.23
+        - name: KUBEVIRT_NUM_NODES
+          value: "3"
         - name: KUBEVIRT_E2E_FOCUS
           value: arm64
         image: quay.io/kubevirtci/bootstrap:v20220906-4f60dd3
```

Comment:
Looking at this tag - it looks to be just built for amd64. I think it would be better to use:
`quay.io/kubevirtci/bootstrap:v20221015-62e6679` which is built for both. 

It might also be worth bumping the unit test job to this tag too.

Reply:
Sorry, I just see your comment.